### DISCUSSION TITLE
Search for POI categories

### DISCUSSION THREAD

**Discussion by kds112** [devemux86/cruiser#236] [Q&A] [answered]
In older Cruiser version there was the possibility to search in a table for a POI category (e.g. fuel station or supermarket) or for all POIs on the map section on your display. After the search the results where marked on the display.
Now I have just one search line and a search for "supermarket" has no results even if there are supermarkets (e.g. Aldi) on the map section. I can't display the POIs on the map (with blue flag) and I can't  tap on this flag to get the details of this POI.
How does it work in the current Cruiser version? How can I look for all POIs in the current map section? Thanks a lot for your help!

**Comment by devemux86** [comment, marked as answer]
@kds112 thanks for the suggestion.

Have you downloaded + activated any offline poi files via menu "Downloads | POI"?
Offline POI are automatically downloaded + activated with the offline maps.

Also are the "Points of interest" selected in the "Settings | Search | Search in"?

Then it should work, as long as the search text exists in the poi tags.
If you have done the above 2 steps, please share a specific example / region to reproduce it.

---

A more friendly **offline** search of POI categories with results on the map is currently being prepared:
- https://github.com/devemux86/cruiser/discussions/195

**Reply by kds112** [reply]
Yes, I have downloaded and activated a POI file (germany from mapsforge) and the selection "search in POI" is set. The problem is, that the string "supermarket" is not included in the poi tag, it is just a category. And in German the name of the category is slightly different ("Supermarkt"). Therefore I am happily waiting for your new offline search feature for POI categories :-) Will it be included in the next Cruiser version?

**Reply by devemux86** [reply]
I meant the OpenStreetMap tag [shop=supermarket](https://wiki.openstreetmap.org/wiki/Tag:shop%3Dsupermarket).

Mapsforge [poi-mapping](https://github.com/mapsforge/mapsforge/blob/master/mapsforge-poi-writer/src/main/config/poi-mapping.xml#L1242-L1244) contains supermarkers, so they should be included in the poi files.
Even if the name in a language is different, the tag remains the same.

> Will it be included in the next Cruiser version?

That's the plan, Cruiser does not take vacations! :beach_umbrella: 

Next Cruiser update is expected to be huge having several big features linked together.
